app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt

```diff
@@ -56,9 +51,6 @@ class ShortcutReceiver @Inject constructor(
         GlobalScope.launch(dispatcher.io()) {
```

Comment:
Consider removing the coroutine launch here, I think we don't need it anymore.

Reply:
Done!